### ISSUE TITLE
Research Github Actions constraints

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#29]
## Acceptance criteria
Evaluation of possibilities:
  - Running Pipeline on every branch/push
  - Running pipeline only on selected branch/push

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
